Assets/MixedRealityToolkit-SDK/Profiles/MixedRealityControllerConfigurationProfile.md

```diff
@@ -0,0 +1,132 @@
+
+## Main Controller Template definition
+---
+![](/External/HowTo/ControllerConfigurationProfile/02-ControllerTemplateDefinition.png)
+
+In the first section of the configuration, the options are detailed as follows:
+
+### Render Motion Controllers
+This defines whether or not controllers should be rendered or not.  By not adding a ControllerVisualizer in to the scene also has the same effect.
+> This also disabled the Global Model options below.
+
+### Use Default Models (for future implementation)
+This option will use the models for the controller direct from the SDK (where available), so that you don't have to configure a custom model.
+
+### Global Left / Right hand models
+These allow you at the top level to define the default model that will be drawn for each controlling hand.  If no hand is available, the Left most model will be used by default.
+
+Additionally, you can provide position and rotational offsets to affect how the model is drawn, to more accurately represent how you want the model drawn based on the controllers position given by the SDK.
+
+> note, Controller rotation offset values are entered as Quaternions for the Alpha.  We may review this going forward.  Use the [this site](https://quaternions.online/) as an aid to turn Euler rotations to Quaternions and ALWAYS remember W = 1 :D
+
+> See the "Example Models" section below for the controller models and their offsets provided in the SDK's "**Standard Assets**" folder.
+
+### Add a New Controller Template
+
+This enables you to add a new controller definition (detailed below) to the profile to add another supported SDK.
+
+## Controller Template
+---
+![](/External/HowTo/ControllerConfigurationProfile/03-ControllerTemplate.png)
+
+Each controller template allows you to configure any of the supported controllers for the various SDK's that have been enabled through the Mixed Reality Toolkit.
+Each controller is added by SDK and the prevailing hand.
+
+> Any SDK Controller types or hands NOT configured will not be detected or used in a running project.
+
+The options for configuring a template are detailed as follows:
+
+### Controller
+Your custom name for the controller, just for easy reference
+
+### Controller Type (Work in progress, see list for enabled devices)
+A drop down list of supported controllers by the Mixed Reality Toolkit, namely:
+
+* Windows Mixed Reality
+    * Motion Controllers
+* OpenVR
+    * Default (fallback) controllers
+    * Oculus Touch
+    * Vive Wand 
+    * Vive Knuckles (experimental as no one has any, using Unity spec)
+* OpenXR
+    * To be supported when there is an OpenXR specification
+* Windows Gaming (*note, these are coming soon*)
+    * Arcade stick
+    * Gamepad
+    * Joystick
+    * Racing Wheel
+* Simulator (coming soon)
+
+### Handedness
+Which hand is configured for this controller definition
+> *Note, Both does not configure all controllers at this time.
```

Comment:
Do we want notes to start with a *? I havent been using one and dont recall see any (many?)

Reply:
removed